pom.xml

```diff
@@ -19,13 +19,20 @@
 		<spotless.version>2.40.0</spotless.version>
 		<swagger.version>2.2.0</swagger.version>
 		<mapstruct.version>1.5.5.Final</mapstruct.version>
+		<hibernate-jpamodelgen.version>6.2.9.Final</hibernate-jpamodelgen.version>
 	</properties>
 
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate.orm</groupId>
```

Comment:
Dodany generator modeli statycznych encji. Przydatny do tworzenia specyfikacji zapytań do bazy.
Jak działa specyfikacja można poczytać tutaj - chyba najlepszy opis.
Metamodel opisany w sekcji: The Generated Metamodel

https://reflectoring.io/spring-data-specifications/